automated_analysis.py

```diff
@@ -465,7 +465,7 @@ def set_survey_percentages(survey_counts, total_survey_counts):
     log.info("Exporting safe to share raw messages for each episode...")
     safe_to_share_messages = []  # of dict of code_string_value to raw messages
     no_of_dns_messages = 0
-    for plan in PipelineConfiguration.RQA_CODING_PLANS[5:]: # Loop through episode 6 -> because previous episodes had been manually processed
+    for plan in PipelineConfiguration.RQA_CODING_PLANS[5:] + PipelineConfiguration.FOLLOW_UP_CODING_PLANS: # Loop through episode 6 -> & followups because previous episodes had been manually processed
         for cc in plan.coding_configurations:
             code_to_messages = OrderedDict()
             for code in cc.code_scheme.codes:
```

Comment:
Suggest changing "Episode" on L492 to something more accurate now it includes follow-ups. Not immediately sure what the best alternative is, but consider "variable", "question", or an alternative of your choosing.

Reply:
Thanks renamed to "Question".